graphbolt/CMakeLists.txt

```diff
@@ -73,6 +73,8 @@ endif()
 add_library(${LIB_GRAPHBOLT_NAME} SHARED ${BOLT_SRC} ${BOLT_HEADERS})
 include_directories(BEFORE ${BOLT_DIR}
                            ${BOLT_HEADERS}
+                           # For CXX20 features.
```

Comment:
could you list what c++20 features are used for clarity?

Reply:
Am I supposed to list them all in here each time I use something new?

Reply:
This PR only uses `std::atomic_ref`